src/www/ui/ui-export-list.php

```diff
@@ -582,6 +593,133 @@ private function printCSV($lines, $uploadtreeTablename, $copyright = false)
+        ],
+        'borders' => [
+            'allBorders' => [
+                'borderStyle' => \PhpOffice\PhpSpreadsheet\Style\Border::BORDER_THIN,
+                'color' => ['argb' => \PhpOffice\PhpSpreadsheet\Style\Color::COLOR_BLACK],
+            ]
+        ],
+        'fill' => [
+            'fillType' => \PhpOffice\PhpSpreadsheet\Style\Fill::FILL_SOLID,
+            'startColor' => ['argb' => '002060'],
+        ],
+    ];
+    $sheet->getStyle('A1:J1')->applyFromArray($styleArray);
+    foreach ($headRow as $key => $val) {
+      $cellName = $key.'1';
+
+      $sheet->getColumnDimension($key)->setWidth($val[0]);
+
+      $sheet->setCellValue($cellName, $val[2]);
+    }
+
+    $styleArray = [
+        'font' => [
+            'color' => ['argb' => '808080'],
+        ],
+        'borders' => [
+            'vertical' => [
+                'borderStyle' => \PhpOffice\PhpSpreadsheet\Style\Border::BORDER_THIN,
+                'color' => ['argb' => 'B2B2B2'],
+            ]
+        ],
+        'fill' => [
+            'fillType' => \PhpOffice\PhpSpreadsheet\Style\Fill::FILL_SOLID,
+            'startColor' => ['argb' => 'FFFFCC'],
+        ],
+    ];
+    $sheet->getStyle('A2:J2')->applyFromArray($styleArray);
+    $annotationRow = array(
+      'A' => array('id', '-'),
+      'B' => array('path', '[Name of the Source File or Path]'),
+      'C' => array('name', '[Name of the OSS used]'),
+      'D' => array('version', '[Version Number of the OSS]'),
+      'E' => array('license', '[License of the OSS. Use SPDX Identifier : https://spdx.org/licenses/]'),
+      'F' => array('download', '[Download URL or a specific location within a VCS for the OSS]'),
+      'G' => array('homepage', '[Web site that serves as the OSSs home page]'),
+      'H' => array('copyright', '[The copyright holders of the OSS. E.g. Copyright (c) 201X Copyright Holder]'),
+      'I' => array('exclude', '[If this OSS is not included in the final version, Check exclude]'),
+      'J' => array('comment','')
+    );
+    foreach ($annotationRow as $key => $val) {
+      $cellName = $key.'2';
+
+      $sheet->setCellValue($cellName, $val[1]);
+    }
+
+    $styleArray = [
+        'borders' => [
+            'allBorders' => [
+                'borderStyle' => \PhpOffice\PhpSpreadsheet\Style\Border::BORDER_THIN,
+                'color' => ['argb' => \PhpOffice\PhpSpreadsheet\Style\Color::COLOR_BLACK],
+            ]
+        ],
+    ];
+
+    $id = 1;
+    foreach ($lines as $row) {
+      $rowNumber = $id + 2;
+      $range = 'A'.$rowNumber.':J'.$rowNumber;
+      $sheet->getStyle($range)->applyFromArray($styleArray);
+      $sheet->setCellValue('A'.$rowNumber, $id);
+      $sheet->setCellValue('B'.$rowNumber, $row['filePath']);
+      if ($row['conclusions'] !== null) {
+        $licenses = implode(',', $row['conclusions']);
+      } else if ($row['agentFindings'] !== null) {
+        $licenses = implode(',', $row['agentFindings']);
```

Comment:
There is a bit of a pickle here. In case the file does not have a license conclusion, or the conclusion is to remove all license found or the conclusion is irrelevant file, the `$row['conclusions']` will be null. In such cases, the licenses found by agents will be printed. I am not sure how to handle it at the moment, maybe there is another column required like the CSV?

Reply:
Also, if we can split the 'License' field in spreadsheet as "scan results" and "concluded results" as in CSV, I think reports will be more consistent.

Reply:
@GMishx , @ag4ums Sorry for late response.
I've split the 'License' field as "scan results" and "concluded results" as suggested.
Thank you for the comment :)

I'll squash the commit once the codes are approved.